src/app/components/Footer/Footer.js

```diff
@@ -140,18 +140,6 @@ const Footer = () => (
 				</Text>
 			</div>
 			<Space block h='64' />
-			<Text>
-				This website uses Google Analytics, their privacy policy can be
-				found{' '}
-				<a
-					target='noreferer'
-					className={cn.mark}
-					href='https://www.google.com/policies/privacy/'
-				>
-					here
-				</a>
-				.
-			</Text>
```

Comment:
Question: do we need to include a similar privacy notice when using Plausible?

Reply:
Nope -> main reason why I picked it. It doesn't use cookies or any advanced tracking. https://plausible.io/data-policy

Reply:
Oh also one last thing, They have an API, so we can create some sort of dash tool